jest/defs.bzl

```diff
@@ -179,21 +227,19 @@ def jest_test(
             colors = colors,
             auto_configure_reporters = auto_configure_reporters,
             auto_configure_test_sequencer = auto_configure_test_sequencer,
-            update_snapshots = update_snapshots,
-            # Tagged manual so it is not built unless the {name}_update_snapshot target is run
-            tags = tags + ["manual"],
+            update_snapshots_mode = update_snapshots_mode,
+            tags = tags + ["manual"],  # tagged manual so it is not built unless the {name}_update_snapshot target is run
             **kwargs
         )
 
         _jest_update_snapshots(
             name = name,
             config = config,
             data = data,
-            tags = tags,
-            update_directory = update_directory,
+            tags = tags + ["manual"],  # tagged manual so it is not built unless run
```

Comment:
Some of the manual tags were missing before? Does this fix some targets being built when not needed?